### PR TITLE
Ignore the .git folder for the dev watcher

### PR DESCRIPTION
**Pull request by Kimtaro**

### CODE REVIEW

File: src/cli/domain/watch.js [hunk 1]
```diff
@@ -14,8 +14,9 @@ module.exports = function(dirs, baseDir, changed) {
       (fileName, currentStat, previousStat) => {
         if (!!currentStat || !!previousStat) {
           if (
-            /node_modules|package.tar.gz|_package|\.sw[op]/gi.test(fileName) ===
-            false
+            /node_modules|package\.tar\.gz|_package|\.sw[op]|\.git/gi.test(
+              fileName
+            ) === false
```
Comment (mattiaerre) [inline src/cli/domain/watch.js]: just a small comment: interesting indentation 😅

Comment (matteofigus) [inline src/cli/domain/watch.js]: is it prettier? :D @nickbalestra can we hack around prettier rules to make that...prettier?

Comment (mattiaerre) [inline src/cli/domain/watch.js]: @matteofigus this is not `prettier` this is `figus-ier` 😄

Comment (Kimtaro) [inline src/cli/domain/watch.js]: I prefer this prettified version over the original since it puts the `===` on the same line as the `false`.

**Comment by matteofigus** [comment]
@Kimtaro thanks for your quick PR 👍 I just published oc@0.37.9 that contains this change

**Pull request merged**